cytoscape-matterjs.js

```diff
@@ -258,14 +257,10 @@
 
       var getPos = function (i, ele) {
         if (ele.parent().length !== 0) {
-          for (var j = 0; j < matterNodes.length; j++) {
-            if (matterNodes[j].shape.id === ele.parent().id()) {
-              return {
-                x: (matterNodes[j].shape.position.x - matterNodes[j].oldX) + nodes[i].position().x,
-                y: (matterNodes[j].shape.position.y - matterNodes[j].oldY) + nodes[i].position().y
-              };
-            }
-          }
+          return {
+            x: (ele.parent().scratch('matter').shape.position.x - ele.parent().scratch('matter').oldX) + nodes[i].position().x,
```

Comment:
Looks good.  It would be great to cache the `.parent()` and `scratch()` calls where possible to avoid unnecessary overhead.